contribution_guide.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to uTensor
+
+Welcome to uTensor contribution guide. Want to make things better? [Talk to us](utensor@googlegroups.com)!
```

Comment:
F the google group, point them to our slack first. utensor.slack.com

Reply:
Phrase it as. Talk to us directly on [Slack](utensor.slack.com) or our [Google Group](utensor@googlegroups.com)